frontend/lab/samples/intro/welcome.htmlf

```diff
@@ -7,12 +7,6 @@
 <p>That's what we'll build up to in this first lab—a bit of a whirlwind tour.</p>
 <h2>🚀 Let's get started!</h2>
 <p>There is also a step-by-step walk through video guide 🎬:</p>
-<iframe
-  src="https://www.youtube-nocookie.com/embed/FBqyiU4iZNY"
-  title="YouTube video player for Introductory Lab Walk Through"
-  frameborder="0"
-  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-  referrerpolicy="strict-origin-when-cross-origin"
-  allowfullscreen
-  class="youtube"
-></iframe>
+<a href="https://youtu.be/FBqyiU4iZNY" target="_blank" class="youtube">
+  <img src="samples/intro/img/youtube.jpeg" alt="YouTube screenshot of walk through" />
```

Comment:
nit: the alt text should perhaps say "Screenshot of YouTube walk through". Same in the .html file

Reply:
thanks, done.